examples/ny-tlc-report-image.yaml

```
@@ -8,7 +8,7 @@ spec:
   version: "1.0"
   # everything under /jobs will be copied to /stackable/spark/jobs
   image: docker.stackable.tech/stackable/ny-tlc-report:0.1.0
-  sparkImage: docker.stackable.tech/stackable/pyspark-k8s:3.3.0-stackable0.0.0-dev
+  sparkImage: docker.stackable.tech/stackable/pyspark-k8s:3.5.0-stackable0.0.0-dev
```

We don't build PySpark images anymore. Python + PySpark is included in the Spark images.
```suggestion
  sparkImage: docker.stackable.tech/stackable/spark-k8s:3.5.0-stackable0.0.0-dev
```